deprecate queryserver-enable-settings-pool flag (#16280)
vitessio/vitess: changelog/21.0/21.0.0/summary.md

|         `emergency_reparent_counts`          |       
|          `planned_reparent_counts`           |      
|      `reparent_shard_operation_timings`      |  

#### <a id="vttablet-flags"/>VTTablet Flags

> - `queryserver-enable-settings-pool` flag was added in v15 have been default on from v17. 

Reviewer: ```suggestion
- `queryserver-enable-settings-pool` flag, added in v15, has been on by default since v17. 
```
